celery/contrib/rdb.py

```diff
@@ -124,6 +124,7 @@ def get_avail_port(self, host, port, search_limit=100, skew=+0):
         this_port = None
         for i in range(search_limit):
             _sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            _sock.setsockopt(socket.SOL_SOCKET, socket.SO_REUSEADDR, 1)
```

Comment:
What happens if SO_REUSEADDR is not supported on the system you are using?
SO_REUSEADDR was added in Linux 3.9 which is fairly recent. Also, it might not be supported on other systems as well.